### PR TITLE
Adds 3 bottles to NT Med.

### PR DESCRIPTION
**Pull request by Xander3359**
## About The Pull Request
Adds Tricordrazine, Isotonic and Paracetamol bottles to the NT Med.
Adds "Chemistry Equipment" to the Valhalla NTMED and adds one to the chemistry area in Valhalla.
## Why It's Good For The Game
These have a packet variant, so it should be fine to add this to the vendor to make chemistry mildly less tedious.
## Changelog
:cl:
add: Tricord, Paracetamol and Isotonic to NTMed
add: NTMED in Valhalla with more stock
/:cl:

### CODE REVIEW

**Comment by TiviPlus** [comment]
@QualityVan

File: code/game/objects/items/reagent_containers/glass/bottle.dm [hunk 1]
```diff
@@ -188,6 +188,18 @@
...
 
+/obj/item/reagent_containers/glass/bottle/paracetamol
+	name = "\improper Paracetamol bottle"
+	desc = "A small bottle. Contains paracetamol - a mild, simple painkiller, good for enduring heavy labor."
+	icon_state = "bottle-1"
+	list_reagents = list(/datum/reagent/medicine/paracetamol = 60)
+
+/obj/item/reagent_containers/glass/bottle/isotonic
+	name = "\improper Isotonic bottle"
+	desc = "A small bottle. Contains tricordrazine - used to aid the body in blood restoration."
```
Comment (QualityVan) [inline code/game/objects/items/reagent_containers/glass/bottle.dm:199]: .

File: code/game/objects/items/reagent_containers/glass/bottle.dm [hunk 2]
```diff
@@ -188,6 +188,18 @@
 	icon_state = "bottle-5"
 	list_reagents = list(/datum/reagent/medicine/tricordrazine = 60)
 
+/obj/item/reagent_containers/glass/bottle/paracetamol
+	name = "\improper Paracetamol bottle"
+	desc = "A small bottle. Contains paracetamol - a mild, simple painkiller, good for enduring heavy labor."
+	icon_state = "bottle-1"
+	list_reagents = list(/datum/reagent/medicine/paracetamol = 60)
+
+/obj/item/reagent_containers/glass/bottle/isotonic
```
Comment (QualityVan) [inline code/game/objects/items/reagent_containers/glass/bottle.dm:197]: Given that the reagent shows up as sal-glu and this is (in principle) for use in chemistry, do you want to label it that way instead? I'm fine with either.

**Review by QualityVan — CHANGES_REQUESTED** [review]
Bottle refilling for chem production is awful and I hate it but it's not dead yet so this is fine, none of these chems are particularly likely to be trouble. What's up with the map changes?

**Review by RipGrayson — APPROVED** [review]

**Review by QualityVan — APPROVED** [review]

**Pull request merged**